runtime/criusupport/criusupport.cpp

```diff
@@ -350,6 +360,29 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 			goto closeWorkDirFD;
 		}
 
+		if (JNI_TRUE == unprivileged) {
+			/*
+			 * dlsym() can return NULL in case of error or if the specified symbol's value is actually NULL.
+			 * In order to distinguish between the two cases we need to check for (and thereby clear) any pending errors by first calling dlerror(),
+			 * then calling dlsym(), then calling dlerror() again to check for any new error. We can then call our function
+			 * via the pointer returned by dlsym(), assuming it is not NULL.
+			 */
+			dlerrorReturnString = dlerror();
+			if (NULL == dlerrorReturnString) {
+				criuSetUnprivileged = (criuSetUnprivilegedFunctionPointerType)dlsym(RTLD_DEFAULT, "criu_set_unprivileged");
+				dlerrorReturnString = dlerror();
+			}
+			if ((NULL == dlerrorReturnString) && (NULL != criuSetUnprivileged)) {
+				systemReturnCode = J9_CRIU_UNPRIVILEGED_NO_ERROR;
+				criuSetUnprivileged(TRUE);
```

Comment:
please  `JNI_FALSE != unprivileged` for consistency

Reply:
Fixed, thanks.